debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/BinlogReaderIT.java

```diff
@@ -484,6 +493,10 @@ public void shouldFailOnUnknownTlsProtocol() {
 
     @Test
     @FixFor("DBZ-1208")
+    @SkipWhenDatabaseVersions({
```

Comment:
Why this format when we are using repeatabel annotation?

Reply:
Just habit, I can repeat them instead if you'd rather.